doc/DOCKER.md

```diff
@@ -114,7 +114,7 @@ $ docker-compose exec web rails c
 Run the testing suite from within the container:
 
 ```
-$ docker-compose exec web rspec spec -fd
+$ docker-compose exec web rspec spec
```

Comment:
Please leave the flag on this example. I think it's helpful for newer developers to be able to view more output.

Reply:
I decided to show both options, with the flag (for the longer screen output) and without the flag (for the shorter and easier-to-read output).